docs/mine/lotus/miner-wallets.md

```diff
@@ -0,0 +1,125 @@
+
+## Control addresses
+
+Control addresses can be used to pay for WindowPoSt messages (ONLY?).
+
+WindowPoSt is the mechanism through which storage is verified in Filecoin and which requires miners to submit proofs for all sectors every 24h. Those proofs are submitted as messages to the chain which pay the respective fees.
+
+Because many other mining related actions require sending messages to the chain, and not all of those are "high value", it may be desirable to use a separate account to send PoSt messages from. This allows for setting lower GasFeeCaps on the lower value messages without creating head-of-line blocking problems for the PoSt messages in congested chain conditions.
+
+To set this up, first create a new wallet, and send it some funds to it for gas fees:
+
+```sh
+lotus wallet new bls
+t3defg...
+
+lotus send t3defg... 100
+```
+
+Next add the control address:
+
+```sh
+lotus-miner actor control set --really-do-it t3defg...
+Add t3defg...
+Message CID: bafy2..
+```
+
+Wait for the message to land on chain:
+
+```sh
+lotus state wait-msg bafy2..
+...
+Exit Code: 0
+...
+```
+
+Finally, check the miner control address list to make sure the address was correctly setup:
+
+```sh
+lotus-miner actor control list
+name       ID      key           use    balance
+owner      t01111  t3abcd...  other  300 FIL
+worker     t01111  t3abcd...  other  300 FIL
+control-0  t02222  t3defg...  post   100 FIL
+```
+
+## Managing balances
+
+To get the balances associated with your miner wallet, run:
+
+```bash
+$ lotus-miner info
+Miner: t01000
+Sector Size: 2 KiB
+Byte Power:   100 KiB / 100 KiB (100.0000%)
+Actual Power: 1e+03 Ki / 1e+03 Ki (100.0000%)
+	Committed: 100 KiB
+	Proving: 100 KiB
+Below minimum power threshold, no blocks will be won
+Deals: 0, 0 B
+	Active: 0, 0 B (Verified: 0, 0 B)
+
+Miner Balance: 10582.321501530685596531 FIL
+	PreCommit:   0.000000286878768791 FIL
+	Pledge:      0.00002980232192 FIL
+	Locked:      10582.321420164834231291 FIL
+	Available:   0.000051276650676449 FIL
+Worker Balance: 49999999.999834359275302423 FIL
+Market (Escrow):  0 FIL
+Market (Locked):  0 FIL
+```
+
+Here, miner id is `t01000`, and it has total balance of `10582.321501530685596531 FIL` with an available balance of `0.000051276650676449 FIL` that can be used as collateral/pay for the pledge. The worker balance is `49999999.999834359275302423 FIL`.
+
+### Adding funds to the owner address
+
+```bash
+./lotus send --from=<worker_address> <miner_id> <amount>
+```
+
+::: tip
+Since the worker address pays for the gas of the send message as well, so don't send the full amount of your worker balance!
+:::
+
+### Withdrawing funds from the miner address
+
+To withdraw funds from the miner address to the worker address you can use the following shortcut:
```

Comment:
The original gist said sending owner balance/rewards to the owner address. That seemed weird. I am assuming it is withdrawing from the owner address to the worker address.

A second assumption is that this is just a shortcut and that the same operation could be performed with `lotus send --from=owner_address workeraddress amount`.